insalan/user/admin.py

```diff
@@ -3,6 +3,24 @@
 from django.contrib.auth.admin import UserAdmin
 from django.contrib.auth.models import Permission
 from .models import User
+from django.contrib.admin import SimpleListFilter
+
+class EmailActivatedFilter(SimpleListFilter):
+    title = 'permissions' # or use _('country') for translated title
+    parameter_name = 'permissions'
+
+    def lookups(self, request, model_admin):
+        return [(True, 'Validé'), (False, 'Non validé')]
```

Comment:
It looks weird in the panel, it's under the section "Permission", "Validé" / "Non Validé". I would pick something like "Courriel validé" / "Courriel non validé"

Reply:
You're right, should be better now.